Kentor.AuthServices/WebSSO/AuthServicesUrls.cs

```diff
@@ -39,6 +39,7 @@ public AuthServicesUrls(HttpRequestData request, ISPOptions spOptions)
         /// </summary>
         /// <param name="applicationUrl">The full Url to the root of the application.</param>
         /// <param name="modulePath">Path of module, starting with / and ending without.</param>
+        [System.Diagnostics.CodeAnalysis.SuppressMessage( "Microsoft.Design", "CA1057:StringUriOverloadsCallSystemUriOverloads" )]
```

Comment:
Looked this up to understand what it warned about. Adding a Justification to the SupressMessage will save others from getting curious (maybe "Incorrect warning. modulePath isn't a string representation of a Uri".)